### DISCUSSION TITLE
Help Wanted

### DISCUSSION THREAD

**Discussion by username-pass** [username-pass/bookmarklets#4] [Announcements]
Requirements:
- knows javascript
- can help with this
- has spare time to work on this
- contributes a "decent" amount (not just changing a letter or something like that)

Reply if you think you can do it.

**Comment by chickennu99et** [comment]
i'll help dont expect much tho(also repl is blocked for me :D)

**Reply by chickennu99et** [reply]
uh how do we communicate

**Reply by username-pass** [reply]
That would be great. You could commit to the GitHub to help. I can communicate through discord, replit, Inject0r chat (Inject0r is the new, better project I am now a part of), GitHub, maybe other things as well. I don't do much here, but I did just figure out how to export all my bookmarklets so there may be a large update soon.
